mlir/utils/jenkins/ci-configs/selected-conv-configs

```diff
@@ -0,0 +1,2 @@
+-n 64 -c 1024 -H 14 -W 14 -k 256 -y 1 -x 1 -p 0 -q 0 -u 1 -v 1 -l 1 -j 1 -m conv -g 1 -t 1
+#-n 256 -c 256 -H 14 -W 14 -k 256 -y 3 -x 3 -p 1 -q 1 -u 1 -v 1 -l 1 -j 1 -m conv -g 1 -t 1
```

Comment:
Why's this commented out?

Reply:
I just want one conv config. Will remove it.